kroxylicious-api/pom.xml

```diff
@@ -162,6 +162,7 @@
                         <breakBuildBasedOnSemanticVersioningForMajorVersionZero>${ApiCompatability.EnforceForMajorVersionZero}</breakBuildBasedOnSemanticVersioningForMajorVersionZero>
                         <excludes>
                             <exclude>io.kroxylicious.proxy.filter.FilterFactoryContext#eventLoop()</exclude> <!-- https://github.com/kroxylicious/kroxylicious/issues/1380 scheduled removal of deprecated API method -->
+                            <exclude>io.kroxylicious.proxy.config.tls.Tls</exclude> <!-- scheduled removal of deprecated API constructor -->
```

Comment:
As @tombentley suggests `<exclude>io.kroxylicious.proxy.config.tls.Tls#Tls(io.kroxylicious.proxy.config.tls.KeyProvider, io.kroxylicious.proxy.config.tls.TrustProvider)</exclude>` works for me to just exclude the constructor and pass the check